hack/kube/overlays/dev/preprocessing-secret.yaml

```diff
@@ -47,3 +47,41 @@ stringData:
     secretKey = "minio123"
     region = "us-west-1"
     bucket = "ais"
+
+    [fileformat]
+    allowlistPath = "/home/preprocessing/.config/allowed_file_formats.csv"
```

Comment:
Should it be `allowListPath`?

Reply:
I'm using "allowlist" as a single word.  I think "allow list path" is confusing because it's not that "list" is being used as a noun and not a verb or an adjective describing "path".